queries/highlights.scm

```diff
@@ -75,6 +75,8 @@
 
 (function parameter: (value_identifier) @parameter)
 
+("unpack") @function.builtin
```

Comment:
> `unpack` is an 'unreserved keyword' (so it doesn’t collide with binding names etc).
https://forum.rescript-lang.org/t/syntax-for-first-class-modules/980/6?u=aspeddro

I'm not sure whether to highlight as `keyword`, `function` or `function.builtin`

Reply:
I have no strong preference here, but I think it is a keyword.

It’s not a function you can compose, pass around, etc. It acts more like the `module(...)` but in contrast to it, the word “unpack” can also be used as a variable name in other contexts. Thanks to Tree-sitter it won’t be highlighted in other contexts, and when we’re indeed unpacking an FCM it behaves like a `module` keyword and should be highlighted similarly.

Reply:
Done 1dcfd0c03b7f82eb49fc4a43b29a153d0b82f189